src/pages/ProductDetail/ProductDetail.js

```diff
@@ -1,8 +1,45 @@
 import React, { useEffect, useState } from 'react';
 import ProductDetailSlider from './ProductDetailSlider';
+import styled from 'styled-components';
+import FavoriteBtn from './FavoriteBtn';
+import FavoriteModal from './FavoriteModal';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faBookmark as farBookmark } from '@fortawesome/free-regular-svg-icons';
 
 const ProductDetail = () => {
   const [productData, setProductData] = useState(null);
+  const [favoriteModal, setFavoriteModal] = useState(false);
+  const productSizes = productData?.size;
+  const productImg = productData?.img[0];
+  const productName = productData?.name;
+  const productInterestedNum = productData?.totalinterestednum;
+
+  const renderInterestedNum = productInterestedNum => {
+    let renderNum = 0;
+    if (productInterestedNum === undefined) {
+      renderNum = 0;
+    }
+    if (productInterestedNum >= 0) {
+      renderNum = productInterestedNum;
+    }
+    if (productInterestedNum >= 1000) {
+      renderNum = productInterestedNum.toLocaleString();
+    }
+    if (productInterestedNum >= 10000) {
+      renderNum = parseFloat((productInterestedNum / 10000).toFixed(1)) + '만';
+    }
+    return renderNum;
```

Comment:
```suggestion
    if (productInterestedNum === undefined) {
      return 0;
    }
    if (productInterestedNum >= 0) {
      return productInterestedNum;
    }
    if (productInterestedNum >= 1000) {
      reutrn productInterestedNum.toLocaleString();
    }
    if (productInterestedNum >= 10000) {
      return parseFloat((productInterestedNum / 10000).toFixed(1)) + '만';
    }

    return 0
```

위와 같은 식으로도 표현할 수 있습니다 참고해주세요~!

Reply:
이부분 궁금했었는데 감사합니다!